3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -22,14 +22,17 @@ return values to pass data back and forth.
 async function fetchData(url) {
   // TODO complete this function
   const response = await fetch(url);
+  if (!response.ok) {
+    throw new Error('Error!!!');
+  }
```

Comment:
The HTTP error you throw here is caught in the catch block (line 66) of the button event handler. This is the expected behaviour although in a real application you would render the error in the HTML ((as you did in the Zodiac app). But the exercise assignment only required you to log to the console, if I'm not mistaken. 

![image](https://user-images.githubusercontent.com/2788771/113438830-a5028180-93e9-11eb-81ea-c1c189c46c1a.png)